pubsub/google/cloud/pubsub_v1/subscriber/_protocol/bidi.py

```diff
@@ -393,6 +393,14 @@ def recv(self):
         return self._recoverable(
             super(ResumableBidiRpc, self).recv)
 
+    @property
+    def is_active(self):
+        """bool: True if this stream is currently open and active."""
+        # Use the operational lock. It's entirely possible for something
+        # to check the active state *while* the RPC is being retried.
+        with self._operational_lock:
+            return self.call is not None and self.call.is_active()
+
```

Comment:
Where is the `is_active` property being used?

Reply:
Down in BackgroundStreamConsumer